### PR TITLE
[update] several edits

### PR DESCRIPTION
**Pull request by gsilvapt**
I did some changes to the layouts and added a few guides. It should solve issue https://github.com/okfn/handbook/issues/90, https://github.com/okfn/handbook/issues/89 and small bits of https://github.com/okfn/handbook/issues/87.

If someone could provide some feedback, I'd appreciate that
- [ ] Images are not displaying: Probably a poor use of Jekyll's `assets`
- [ ] Current structure does not please me - have to think about how to improve it
- [ ] Forgot to add how to make Pull Requests in `working-with-git.md` to this PR
- [ ] Missing table of contents (don't know how to do it)
- [ ] Menu needs updates (as per https://github.com/okfn/handbook/issues/91)

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 title: Data Patterns
-short_title: Data Patterns
-description: "A collection of tips, tricks and patterns for data work."
+short_title: Labs Handbook
+description: "A collection of guides, counselling and other advise to participate in OKFN Labs"
```
Comment (rufuspollock) [inline _config.yml:3]: We never use "OKFN" anymore except in URLs. Should always be "Open Knowledge Labs" ;-)

Comment (rufuspollock) [inline _config.yml:3]: "counselling" is odd phrasing. I would just have:

"Guides and advice on participating in Open Knowledge Labs and its projects"

File: core-guides/core-data-curators.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+1. Follow the [GitHub's issue tracker](https://github.com/datasets/registry/issue) and look for a package with 3-star priority, or
+
+2. Get your own data and start preparing it.
+
+Which one you should go to first is really up to you. The difference, however, is that some are more needed than others and 
+you could use that knowledge and skill to tackle on some of the project's priorities.
+
+Other than that, the workflow and requirements is pretty much the same.
+
+## Following urgencies
```
Comment (rufuspollock) [inline core-guides/core-data-curators.md:23]: Odd phrasing in english. Not quite sure what you mean by "urgencies"

File: core-guides/core-data-curators.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+# Getting data
+
+The first step is, naturally, finding data to work with. There are two common ways you can do this:
+
+1. Follow the [GitHub's issue tracker](https://github.com/datasets/registry/issue) and look for a package with 3-star priority, or
+
+2. Get your own data and start preparing it.
+
+Which one you should go to first is really up to you. The difference, however, is that some are more needed than others and 
+you could use that knowledge and skill to tackle on some of the project's priorities.
```
Comment (rufuspollock) [inline core-guides/core-data-curators.md:19]: line-breaking.

Recommend we do _not_ do line breaks - a single paragraphy _is_ a single line.

This is a very minor point and we can correct later - but good to know.

File: core-guides/core-data-curators.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+## Your Own Taste
+
+You can also search for topics of your preference, or even suggest them in the issues page if they are not there already.
+
+
+# Preparing the Data
+
+Now, back to the workflow. This is the troublesome part. The job is to, first, find the best and most reliable source, convert 
+if to CSV or XLSX and then work on preparing the data package. If you do not know what `CSV` is, we advise you to read the
```
Comment (rufuspollock) [inline core-guides/core-data-curators.md:37]: We never want XLSX I think.

File: core-guides/core-data-curators.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+You can also search for topics of your preference, or even suggest them in the issues page if they are not there already.
+
+
+# Preparing the Data
+
+Now, back to the workflow. This is the troublesome part. The job is to, first, find the best and most reliable source, convert 
+if to CSV or XLSX and then work on preparing the data package. If you do not know what `CSV` is, we advise you to read the 
+[Data Guides](data-guides/).
+
+## Convert to .CSV
```
Comment (rufuspollock) [inline core-guides/core-data-curators.md:40]: Generall we want this scripted.

File: core-guides/core-data-curators.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+
+As mentioned before, the first step is to have a source `csv` file and you can do that, for instance, by click in the right 
+tab in the [World Bank of Data](http://data.worldbank.org/). See picture below:
+
+![image]({{ site.url }}/images/export.jpg)
+
+Chances are, many world data references have a similar option.
+
+## Directory structure
```
Comment (rufuspollock) [inline core-guides/core-data-curators.md:49]: To what extent are we duplicating general guide to creating a data package. Not a problem right now - let's get this in but worth keeping in mind.

File: core-guides/core-data-curators.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+data:
+      python $(SCRIPTDIR)/process.py
+
+      clean:
+            rm -f $(DATADIR)/*
+
+            .PHONY: all data clean
+```
+
+## The ugly job of quality assurance
```
Comment (rufuspollock) [inline core-guides/core-data-curators.md:97]: Let's keep these titles simpler and with less commentary ;-)

e.g. this could just be:

"Quality Assurance"

File: core-guides/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+[This video](https://vimeo.com/133587259) will help you understand the core functionalities of the platform where we host 
+the forums. Worry not!, you will find most of them very intuitive and will learn along the way. In fact, feel free to ask for 
+help in there if you feel lost.
+
+In order to keep things organized in GitHub, we use the label system. They are very straightforward and easy to understand. 
+It is important that you respect those, that are usually attributed by Core Datasets Managers. This group of people keeps 
+track of the project's needs and tries to forward the project's direction accordingly.
+
+Following the project's direction
```
Comment (rufuspollock) [inline core-guides/getting-started.md:19]: For markdown we want to use `##` method for doing headings rather than underling.

File: data-guides/appendix/glossary.md [hunk 9]
```diff
@@ -0,0 +1,174 @@
+---
```
Comment (rufuspollock) [inline data-guides/appendix/glossary.md:1]: Where did this glossary come from? Was this not already inside patterns?

File: core-guides/working-with-git.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
+---
+title: Working with Git
```
Comment (rufuspollock) [inline core-guides/working-with-git.md:2]: I am pretty sure there are other guides online on this e.g. help.github.com. My intuition here would be having something short here and largely referencing other guides would be best - we run the risk of going out of date and this is not the core thing we are doing (e.g. comments on Ubuntu vs Windows users).

File: data-guides/index.md [hunk 11]
```diff
@@ -1,20 +1,28 @@
 ---
-title: Welcome to the Data Wrangling Handbook!
+title: Welcome to the Labs Handbook!
 ---
 
-The Data Wrangling Handbook is a companion text to the [`School of Data`](http://schoolofdata.org/). Its function is something like a traditional textbook - it will provide the detail and background theory to support the School of Data courses and challenges.
+The Labs Handbook is a companion text to the [`School of Data`](http://schoolofdata.org/). Its function is something like a
```
Comment (rufuspollock) [inline data-guides/index.md:5]: This is no longer true and we should delete this paragraph (though we may want to reference the School of Data courses somewhere).

File: data-guides/intro.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (rufuspollock) [inline data-guides/intro.md:1]: This stuff does not make so much sense now we have moved out of patterns. The general move of stuff out of patterns is an example of a major structural change we _should_ have discussed in a github issue first before implementing.

File: index.html [hunk 13]
```diff
@@ -9,33 +9,32 @@
   <section class="banner" data-adaptive-background='1' data-ab-css-background='1'>
     <div class="wrapper">
       <h1>{{ site.title }}</h1>
-      <p>{{ site.description }}</p>
+        <p>{{ site.description }}</p>
```
Comment (rufuspollock) [inline index.html:12]: Why have you changed indentation here?

Also this is a good example of a change that should _definitely_ have been a separate commit and quite possibly a separate Pull Request. Right now as reviewer i have only an all or nothing choice to merge or not merge this PR ;-)

**Comment by rufuspollock** [comment]
@gsilvapt really good work. Some comments inline. General comments:
- More meaningful commit messages e.g.`[update]: several edits` does not tell me much suggest: `[data][xl]: add several data guides and move all patterns under data-guides/` (note use of "size" descriptor for commit in second bracket `[xs,s,m,l,xl]` are the options).
- Slightly less in a single PR would be good -- at least at this early stage -- so I can review, we can make any adjustments and then do the bigger item.
- I am not sure about relocating all the patterns stuff. This is the kind of thing I really recommend discussing in an issue before just doing. Right now I'll merge but we may need to think about moving it back.
- All the checklist "issue" items might be better off in a specific issue (given the number and substance of them)

File: core-guides/core-data-curators.md (hunk 3 above)
Comment (gsilvapt) [inline core-guides/core-data-curators.md:19]: I think it doesn't change the outlook but I also did that because it is easy to scroll through text in vim. I can fix it though.

File: core-guides/core-data-curators.md (hunk 4 above)
Comment (gsilvapt) [inline core-guides/core-data-curators.md:37]: The audience I though for these guides would be non-power users. To me, I can simply download data from the web, whether it is excel, csv or an API. But for a regular user, perhaps things are not that straightforward. I can remove it if you think it is better ;)

File: core-guides/core-data-curators.md (hunk 6 above)
Comment (gsilvapt) [inline core-guides/core-data-curators.md:49]: Yeap, I mentioned this in some issues. I think I did not understand very clearly what are the purposes of both guides. Sorry for that!

File: core-guides/working-with-git.md (hunk 10 above)
Comment (gsilvapt) [inline core-guides/working-with-git.md:2]: Absolutely right. That will be my priority right now. :+1:

File: data-guides/appendix/glossary.md (hunk 9 above)
Comment (gsilvapt) [inline data-guides/appendix/glossary.md:1]: I didn't make any changes in `/data-guides/` folder, other than changing its name. What are these or where they stand, I do not know exactly.

Should I move this somewhere?

File: data-guides/index.md (hunk 11 above)
Comment (gsilvapt) [inline data-guides/index.md:5]: I'd suggest in the Data-wrangling and tech guides.

File: data-guides/intro.md (hunk 12 above)
Comment (gsilvapt) [inline data-guides/intro.md:1]: I saw this file, didn't make much sense but it was already there. I just left it in case there is something important here to keep.

File: index.html (hunk 13 above)
Comment (gsilvapt) [inline index.html:12]: Potentially a mistake I didn't realize at the time. Sorry!

File: core-guides/core-data-curators.md (hunk 4 above)
Comment (rufuspollock) [inline core-guides/core-data-curators.md:37]: The point is we won't be converting to XLSX. Downloading as XLSX is fine but converting to it: no.

File: core-guides/core-data-curators.md (hunk 3 above)
Comment (gsilvapt) [inline core-guides/core-data-curators.md:19]: Ahh, I have it working as normal in mu blog. Fixing soon.
